expyfun/stimuli/_tracker.py

```diff
@@ -0,0 +1,862 @@
+"""Adaptive tracks for psychophysics (individual, or multiple randomly dealt)
+"""
+# Author: Ross Maddox <ross.maddox@rochester.edu>
+#
+# License: BSD (3-clause)
+
+import numpy as np
+import time
+from scipy.stats import binom
+import json
+import matplotlib.pyplot as plt
```

Comment:
I did that in a commit that is still on my local system and it doesn't fix it. I'll push it so you can see.